content/migration-from-otrs-6.rst

```diff
@@ -28,9 +28,9 @@ With the OTOBO Migration Interface it is possible to perform the following migra
 
 3. A general migration where many combinations are possible.
 
-    1. Change server and OS: Migrate and simultaneously move to a new application server and operating system.
+    1. Change server: Migrate and simultaneously move to a new application server.
 
-    2. Separate servers: It does not matter whether your OTRS/ ((OTRS)) Community Edition was previously installed on two separate servers (application and database servers). It also does not matter whether you want to change OTOBO to such a configuration.
+    2. Separate servers: It does not matter whether your OTRS/ ((OTRS)) Community Edition was previously installed on two separate servers (application and database server). It also does not matter whether you want to change OTOBO to such a configuration.
```

Comment:
Bin wieder drüber gestolpert. Vielleicht so?
Separate servers: Regardless of the prior setup of your OTRS / ((OTRS)) Community Edition on one or two servers (application and database server), you are free to change this configuration for OTOBO.

Reply:
Ich habe es neu, etwas positiver, formuliert,